FRONTEND/src/routes/index.jsx

```diff
@@ -34,9 +35,10 @@ const Routing = () => {
             <Route path="/error" element={<ErrorPage/>}/>
             <Route path='/howitworks' element={<Howitworks />} />
             <Route path='/privacy' element={<Privacy />} />
+            <Route path='/Gallery' element={<Gallery/>} />
             {/* Protect this route or any other routes that require authentication */}
             {/* <Route 
-              path='/media-owners' 
+              path='/media-ownegrs' 
```

Comment:
Why play around with path!?

Reply:
updated

Reply:
???? that  can be typed error and missed to resovled beacuse that is commented part